gpMgmt/bin/gplogfilter

```diff
@@ -322,6 +322,34 @@ def openOutputFile(ifn, zname, options):
 
     return fileOut, filesToClose
 
+def shouldSkipFile(ifn, begin, end):
+    # case from openInputFile
+    if os.path.isdir(ifn):
+        return False
+
+    # matchedGroup in case of match would be not None and contains next groups:
+    # - group(0) - whole file name
+    # - group('datetime') - datetime of log (but may be without time suffix)
+    # - group('time') - not None if time suffix exists
+    matchedGroup = re.match(
```

Comment:
Regular expressions in Python are not very fast, so it is better to compile outside the function and use it inside.